server/option.go

```diff
@@ -210,7 +218,7 @@ func (o *scheduleOption) reload(kv *core.KV) error {
 		namespaces[name] = *ns.load()
 	}
 	cfg := &Config{
-		Schedule:      *o.load(),
+		Schedule:      *o.load().clone(),
```

Comment:
is that necessary? `*` will lead to deep copy

Reply:
it's necessary,  the slice not deep copy in default.